src/Tokenizer/Token.php

```diff
@@ -20,12 +20,21 @@
  * Representation of single token.
  * As a token prototype you should understand a single element generated by token_get_all.
  *
+ * @phpstan-type _TokenKind int|string
```

Comment:
you can re-use https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8925/commits/598fc90aa6096b23200e94592ac44bf93a4692eb#diff-a4ce6ffd8cb697aecebba7af3c784ed29078f645544c069ec7fecbe9417e2a04R22